.circleci/config.yml

```diff
@@ -129,8 +133,8 @@ jobs:
           name: Install PyTorch
           command: python -m pip install torch==1.8.1+cu102 torchvision==0.9.1+cu102 -f https://download.pytorch.org/whl/torch_stable.html
       - run:
-          name: Install psutil
-          command: python -m pip install psutil
+          name: Install psutil and ninja
```

Comment:
May leave a comments to indicate that we install ninja to accelerate compilation?

Reply:
got it